api/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'war'
+}
```

Comment:
It is a good practice to extract the same dependencies and plugins into root build.gradle and apply them to all submodules(please recall the difference between allprojects and subprojects in gradle) like:
```
submodules {
 apply plugin: 'java'

delendencies {
...}
}
```

In addition to this change please extract at least spring version into variable, e.g
```
ext {
    springVersion = "5.2.1"
}
```